scripts/install.sh

```diff
@@ -88,7 +88,13 @@ esac
 echo "* Create distribution directories..."
 mkdir -p "$maestro_tmp_folder"
 
-download_url="https://github.com/mobile-dev-inc/maestro/releases/latest/download/maestro.zip"
+
+if [ -z "$MAESTRO_VERSION" ]; then
+    download_url="https://github.com/mobile-dev-inc/maestro/releases/latest/download/maestro.zip"
+else
+    download_url="https://github.com/mobile-dev-inc/maestro/releases/download/cli-$MAESTRO_VERSION/maestro.zip"
```

Comment:
Anticipating a question of why I don't just assign a default value to `MAESTRO_VERSION` - the URLs have swapped order depending on whether you are fetching a latest version or specific version:

```
.../latest/download/...
```

and

```
.../download/{version}/...
```

🤷